docs/config/README.md

```diff
@@ -404,6 +404,10 @@ The module will be shown if any of the following conditions are met:
 | `disabled`           | `false`    | Disables the `python` module.                                               |
 | `pyenv_version_name` | `false`    | Use pyenv to get Python version                                             |
 | `pyenv_prefix`       | `"pyenv "` | Prefix before pyenv version display (default display is `pyenv MY_VERSION`) |
+<<<<<<< HEAD
+=======
+
+>>>>>>> Set default prompt character to ❯
```

Comment:
Looks like there was a bit of merge issue here haha

Reply:
Ugh, and over whitespace as well :|

Thanks for the catch.